packages/.prettierignore

```diff
@@ -0,0 +1 @@
+**/**/*.md
```

Comment:
do we really want to entirely ignore `.md` files? I made the command prettier-readmes so that we can improve the readmes manually.

Reply:
I copied it from the original `package.json`, where it was originally.

```json
  "lint-staged": {
    "linters": {
      "*.{js,jsx,md,ts,tsx}": [
        "prettier --write packages/{bundle,component,core,embed,playground}/src/**/*.{js,jsx,ts,tsx}",
        "prettier --write __tests__/**/*.{js,jsx,ts,tsx}",
        "git add"
      ]
    },
    "ignore": [
      "**/**/*.md"
    ]
  },
```

After upgrading `lint-staged`, it removed `ignore` options and suggested to move it to `.prettierignore` instead. See it here, https://github.com/okonet/lint-staged#ignoring-files.

Reply:
I know you have Prettier for Markdown and looks weird to me too.

But looks like before my bump, Prettier is disabled for Markdown.

Reply:
this ignores prettier for the lint-staged command, which happens on commit. I'm just not sure if .prettierignore conflicts with the manual command `npm run prettier-readmes`

Reply:
Oh sorry, didn't know there is another NPM script.

I should remove `.prettierignore` and let Husky run on paths without Markdown. Hold on.

Reply:
can you also add the ignore back to the package.json files?

Reply:
I can't, the option is deprecated from `lint-staged`. That's why the recommend using `.prettierignore`.

Reply:
ahhhh